packages/components/components/elvis-toolbox/src/hooks/useFocusTrap.ts

```diff
@@ -65,15 +62,21 @@ export const useFocusTrap = (): {
         lastItem.addEventListener('keydown', handleLastItemTab);
       }
     });
+
+    mo = new MutationObserver(() => {
+      releaseFocusTrap();
+      trapFocus(focusTrapContainer);
+    });
```

Comment:
This assumes that all the content is new. If we had a list of buttons and later added a new button, would we have to TAB through the whole list again? 🤔

Reply:
Hmm thats true, this will reset the focus 🤔 That's not ideal

Reply:
I've now addressed this issue